packages/frontend/src/pages/Send/L1CanonicalBridgeOption.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Token } from '@hop-protocol/sdk'
+import { BigNumber } from 'ethers'
+import { Div, Flex } from 'src/components/ui'
+import { toTokenDisplay } from 'src/utils'
+import Network from 'src/models/Network'
+import CanonicalBridge from 'src/models/CanonicalBridge'
+import { RadioButtonChecked, RadioButtonUnchecked } from '@material-ui/icons'
+
+interface Props {
+  amount?: BigNumber
+  destNetwork?: Network
+  destToken?: Token
+  estimatedReceivedDisplay?: string
+  l1CanonicalBridge?: CanonicalBridge
+  sendL1CanonicalBridge?: () => void
+  setUl1cb: (u: boolean) => void
```

Comment:
this callback name `setUl1cb` isn't clear what it's for, the name can be more descriptive

Reply:
i'll go with `setUsingNative bridge`